### DISCUSSION TITLE
Classes extraction when there are no quotes

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#10834] [Help]
Hi all,

is tailwind able to extract class names if there are no quotes around them?

**Comment by wongjn** [comment]
Yep!

**Reply by ghost** [reply]
Thank you!
Do you see any reason it's not able to extract `bg-gray-600` from this line:
`     active_option: ~W(text-white bg-gray-600),`
?

**Reply by wongjn** [reply]
Several things come to mind:
- The file that this code is in may not be covered by the `content` file globs in the Tailwind configuration.
- `bg-gray-600` may not be a class name that Tailwind would generate a CSS rule for.
- Perhaps the parenthesises or other language tokens around it could be tripping up the Tailwind string extractor.

**Reply by ghost** [reply]
>     * The file that this code is in may not be covered by the `content` file globs in the Tailwind configuration.

I tried modifying the line in the file with `     active_option: ~W(text-white "bg-gray-600"),` and it extracts correctly

>     * `bg-gray-600` may not be a class name that Tailwind would generate a CSS rule for.

The test I made above covers this concern

>     * Perhaps the parenthesises or other language tokens around it could be tripping up the Tailwind string extractor.

Should I open an issue?

**Reply by wongjn** [reply]
You could try opening an issue. You can also try [customizing the extraction logic](https://tailwindcss.com/docs/content-configuration#customizing-extraction-logic).
